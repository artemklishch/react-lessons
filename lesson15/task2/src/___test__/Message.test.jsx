import React from 'react';
import { shallow } from 'enzyme';
import Message from '../Message';

describe('<Message />', () => {
  it ('should render component with text', () => {
    const props = {
      text: 'Hello world!'
    }
    const wrappedComponent = shallow(<Message {...props} />);
    expect(wrappedComponent).toMatchSnapshot();
  })

  it ('shouldn`t render component with text', () => {
    const props = {
      text: ''
    }
    const wrappedComponent = shallow(<Message {...props} />);
    expect(wrappedComponent).toMatchSnapshot();
  })
});