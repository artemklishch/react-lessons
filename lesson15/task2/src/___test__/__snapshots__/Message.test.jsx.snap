// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Message /> should render component with text 1`] = `
<div
  className="message"
>
  Hello world!
</div>
`;

exports[`<Message /> should render text 1`] = `
<div
  className="message"
>
  Hello world!
</div>
`;

exports[`<Message /> should render text 2`] = `""`;

exports[`<Message /> shouldn\`t render component with text 1`] = `""`;
